ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>11.0</string>
+  <string>12.0</string>
```

Comment:
Curious why this needed to bump?

Reply:
Looks like it's the minimum for xcode 15.4, which is what I'm using.

<img width="945" alt="image" src="https://github.com/user-attachments/assets/6a9c94aa-8507-400d-96d5-7eb3c96a8ce3">

https://developer.apple.com/support/xcode/

Reply:
Did some Googling and iOS 11 looks dead as a doorknob. I think this is fine.